client/lib/checkout/payment-methods.tsx

```diff
@@ -131,6 +139,9 @@ export const PaymentMethodSummary = ( {
 	if ( type === PARTNER_PAYPAL_EXPRESS ) {
 		return <>{ email || '' }</>;
 	}
+	if ( type === PARTNER_RAZORPAY ) {
+		return <>{ translate( 'Razorpay' ) }</>;
```

Comment:
@kanishkdudeja @billrobbins should this be `Unified Payments Interface (UPI)` or maybe `Razorpay UPI`?  I don't know which would be the more common way to refer to it in India, but although Stripe is pretty common, we tend to leave the processor out of this label. The icon helps, but we should assume that it might not be shown.

Reply:
I'll defer to @kanishkdudeja since he has the best context on this.

Reply:
I personally lean towards `Unified Payments Interface (UPI)` since I'm presuming users don't really remember which payment gateway they initially used to set up the subscription.